packages/grafana-api-clients/src/index.ts

```diff
@@ -1,2 +1,47 @@
+import { generatedAPI as advisorAPIv0alpha1 } from './clients/advisor/v0alpha1';
+import { generatedAPI as dashboardAPIv0alpha1 } from './clients/dashboard/v0alpha1';
+import { generatedAPI as folderAPIv1beta1 } from './clients/folder/v1beta1';
+import { generatedAPI as iamAPIv0alpha1 } from './clients/iam/v0alpha1';
+import { generatedAPI as migrateToCloudAPI } from './clients/migrate-to-cloud';
+import { generatedAPI as playlistAPIv0alpha1 } from './clients/playlist/v0alpha1';
+import { generatedAPI as preferencesUserAPI } from './clients/preferences/user';
+import { generatedAPI as preferencesAPIv1alpha1 } from './clients/preferences/v1alpha1';
+import { generatedAPI as provisioningAPIv0alpha1 } from './clients/provisioning/v0alpha1';
+import { generatedAPI as rulesAPIv0alpha1 } from './clients/rules.alerting/v0alpha1';
+import { generatedAPI as shortURLAPIv1alpha1 } from './clients/shorturl/v1alpha1';
+// PLOP_INJECT_IMPORT
+
+/** RTK Query middleware for all API clients  */
+export const allMiddleware = [
```

Comment:
```suggestion
export const generatedMiddleware = [
```
I think `allMiddleware` is not really descriptive, same for the reducers. Or maybe it'd be `rtkMiddleware` 🤔